code/modules/reagents/chemistry/holder.dm

```diff
@@ -1709,6 +1711,8 @@
 					tooltip_bool = TRUE
 			data["reagent_mode_recipe"]["catalysts"] += list(list("name" = reagent.name, "id" = reagent.type, "ratio" = reaction.required_catalysts[reagent.type], "color" = color_r, "tooltipBool" = tooltip_bool, "tooltip" = tooltip))
 		data["reagent_mode_recipe"]["isColdRecipe"] = reaction.is_cold_recipe
+	else
+		data["reagent_mode_recipe"] = null
```

Comment:
Doesn't `= null` not actually do anything with tgui if it was already passed previously? I swore it patched on, CC @stylemistake if I'm going crazy.

Reply:
assigning null does change it to null

Reply:
weeeird, wonder what im thinking of then